src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/target.py

```diff
@@ -133,8 +143,16 @@ def _create_response_object(self, responses):
                         str(response.get('Id'))
                     )
                 )
-        if accounts_found == 0:
-            raise NoAccountsFoundError(f"No accounts found in {self.path}")
+
+        if self.allow_empty_deployment_maps == "TRUE":
+            # We now allow empty deployment maps
+            if accounts_found == 0:
+                LOGGER.info(
+                    'Create_response_object: AWS accounts found is 0',
+                )
+        else:
+            if accounts_found == 0:
+                raise NoAccountsFoundError(f"No accounts found in {self.path}")
```

Comment:
To apply what @StewartW commented in a suggestion:
```suggestion
        if accounts_found == 0 and not self.allow_empty_deployment_maps:
            raise NoAccountsFoundError(f"No accounts found in {self.path}")
```